test/test_main.py

```diff
@@ -206,8 +204,6 @@ def test_multiple_events_handled(self):
             'USE_ACTIONS': True,
         }))
 
-        fake_script(self, 'action-get', "echo '{}'")
```

Comment:
It seems surprising to me that the only test changes this causes is *removing* tests?
I suppose there may not be a visible change, as before you should be able to read params, and after you should be able to read the correct params.

Reply:
Addressed in https://github.com/canonical/operator/pull/132/commits/61364bbf52b400d1a385f034046a4a84f7cdbf76